processor/resourcedetectionprocessor/README.md

```diff
@@ -17,6 +17,18 @@ details of which are currently pending confirmation in the OpenTelemetry specifi
     * host.name
     * os.type
 
+* System detector custom configuration example:
```

Comment:
```suggestion
By default `host.name` is being set to FQND if possible, and a hostname provided by OS used as fallback.
This logic can be changed with `hostname_sources` configuration which is set to `["dns", "os"]` by default.

Use the following config in order to use hostname provided by OS only: 
```

Reply:
I have added the suggested changes. Please, review the same.